atomic_reactor/plugins/pre_inject_yum_repo.py

```diff
@@ -65,10 +93,21 @@ def run(self):
         self.log.info("creating directory for yum repos: %s", host_repos_path)
         os.mkdir(host_repos_path)
 
-        for repo, repo_content in self.workflow.files.items():
-            yum_repo = YumRepo(repourl=repo, content=repo_content, dst_repos_dir=host_repos_path,
-                               add_hash=False)
-            yum_repo.write_content()
+        for repo_filename, repo_content in self.workflow.files.items():
+            # Update every repo accordingly in a repofile
+            # input_buf ---- updated ----> updated_buf
+            with StringIO(repo_content) as input_buf, StringIO() as updated_buf:
+                for line in input_buf:
+                    updated_buf.write(line)
+                    # Apply sslcacert to every repo in a repofile
+                    if line.startswith('['):
```

Comment:
I still think that there can be spaces before that, so to be sure, I'd just lstrip it